homeassistant/components/neato/sensor.py

```diff
@@ -81,6 +81,9 @@ def available(self):
     @property
     def state(self):
         """Return the state."""
+        if not self._state:
```

Comment:
I don't think this is not necessary as we won't query state if `available` returns `False`.

Reply:
Done. Thanks